docs/001_develop/03_client-capabilities/019_styling/02_direct-customization-genesis-components/03_design-system-configurator.mdx

```diff
@@ -0,0 +1,125 @@
+---
+title: 'Design system configurator'
+sidebar_label: 'Design system configurator'
+id: styling-direct-dsc
+keywords: [extend component, create component, styling]
+tags:
+- styling
+- extend component
+- create component
+- styling
+sidebar_position: 3
+---
+
+The Design System Configurator can be used to customise and tailor the design system according to your specific needs. This
+configures many of the [CSS variables](../styling-direct-css-vars/) that the `rapid` design system uses while showing
+some example components live update to your changes.
+
+
+To run it you can navigate to the `/client` directory in your app created via Genesis Create or `genx` and run
+```bash
+npm run dsconfig
+```
+or run that npm job via your IDE.
+
+On the left-hand side, is the [Editor](#editor), where you can make all the modifications you want. On the right side, you can see the [Preview](#preview) with the results of those changes.
+
+Try changing the configurations and observe the reflected changes on the right.
+
+<div class="demo-view">
+  <img
+    id="shadowed-img"
+    src={require("@site/static/img/DSC-checkbox.PNG").default}
+    alt="Image capture of DSC app"
+  />
+  <div>
+    <!-- this link only works when deployed -->
+    <a class="overlay-link" href="/docs/dsc/" target="_blank">
+      Launch DSC &#x2197;
+    </a>
+  </div>
+</div>
+
+:::tip
+When using the DSC on your application it configure the `rapid` design system, which is the default. It can additionally be used to configure design
```

Comment:
```suggestion
When using the DSC on your application it configures the `rapid` design system, which is the default. It can additionally be used to configure design
```